man/man5/vdev_id.conf.5

```diff
@@ -92,6 +92,11 @@ before a generic mapping for the same slot.
 In this way a custom mapping may be applied to a particular channel
 and a default mapping applied to the others.
 .
+.It Sy zpad_slot Ar digits
+Pad slot numbers with zeros to make them
+.Ar digits
+long.
+.
```

Comment:
Might want to add another line:
```
This padding can be useful when using multi-actuator drives.
```

Reply:
Yes, it is definitely useful with many multi-actuator drives but not unique to just that use. I added a bit to the description to explain the purpose.

> +Pad slot numbers with zeros to make them
> +.Ar digits
> +long, which can help to make disk names a consistent length and easier to sort.